x-pack/platform/plugins/shared/alerting/server/routes/alert_delete/transforms/transform_last_run_response/v1.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AlertDeleteLastRun } from '@kbn/alerting-types';
+import type { AlertDeleteLastRunResponseV1 } from '../../../../../common/routes/alert_delete';
+
+export const transformAlertDeleteLastRunToResponse = ({
```

Comment:
It's a nit because the logic is so small, but we could have a test file for this.

Reply:
💔